<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BackgroundChannelSetting</class>
 <widget class="QWidget" name="BackgroundChannelSetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>180</width>
    <height>213</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QCheckBox" name="bg_channel_checkBox">
         <property name="text">
          <string>Background channel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Color</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QComboBox" name="bg_channel_comboBox"/>
       </item>
       <item>
        <widget class="QPushButton" name="color_pushButton">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="show_on_top_checkBox">
     <property name="text">
      <string>Display on top</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="ref_groupBox">
     <property name="title">
      <string>Reference</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="select_reference_label">
        <property name="text">
         <string>Select channel.</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="select_reference_comboBox"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="filter_groupBox">
     <property name="title">
      <string>Filter raw</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="filter_low_label">
          <property name="text">
           <string>Low cutoff</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="filter_low_doubleSpinBox">
          <property name="maximum">
           <double>30000.000000000000000</double>
          </property>
          <property name="value">
           <double>250.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="filter_high_label">
          <property name="text">
           <string>High cutoff</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="filter_high_doubleSpinBox">
          <property name="maximum">
           <double>30000.000000000000000</double>
          </property>
          <property name="value">
           <double>6000.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="apply_pushButton">
     <property name="text">
      <string>Apply Changes</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>bg_channel_checkBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>bg_channel_comboBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>93</x>
     <y>15</y>
    </hint>
    <hint type="destinationlabel">
     <x>189</x>
     <y>26</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
